zds/member/models.py

```diff
@@ -378,6 +378,20 @@ def get_hats(self):
         hats.sort(key=lambda hat: hat.name)
         return hats
 
+    def get_requested_hats(self):
+        """
+        Return all current hat requests by this user.
```

Comment:
Prosition : Return all current hat requests by this user. --> Return all current hats requested by this user.